src/mono/browser/runtime/strings.ts

```diff
@@ -45,7 +45,7 @@ export function stringToUTF8(str: string): Uint8Array {
 }
 
 export function stringToUTF8Ptr(str: string): CharPtr {
-    const bytes = str.length * 2;
+    const bytes = (str.length + 1) * 2;
```

Comment:
+1 is null terminator? Why till now we did not reserve space for it and now we're starting?

Reply:
yes, it didn't fail so far because nobody used many multi-byte characters yet.